### ISSUE TITLE
Passing array as a log message

### ISSUE THREAD

**Issue by mikemadisonweb** [mitoeurope/yii2-sentry#9]
Hi guys,
I think you should make it clear in docs that if you pass array as a log message(which is fine according to Yii docs) you should set key 'msg'. Otherwise it will fail silently at line:
```
$data['message'] = substr($data['message'], 0, $this->message_limit);
```
Although file targets will process such message without any trouble, because of 'formatMessage' method.

**Comment by nkovacs** [comment]
The issue is in https://github.com/hellowearemito/yii2-sentry/blob/1.0.0-dev/src/Target.php#L66

This branch assumes that the message (`$context`) is a string, but it might be an array without a `msg` key.
I'm not sure what we should do here. We could use `VarDumper::export()` like the other log targets.

**Comment by mikemadisonweb** [comment]
It may be an option, but as i tried it, array message doesn't look pretty in sentry and also it get truncated. As you can see in line i've posted from Raven, there is a message length limit and by default it is set to 1024 symbols.
I think it is fine that target accept 'msg' key in array and put all other array contents in 'extra' field, but it should be in the docs. Also it would be nice to run VarDumper::export() if array does not have 'msg' key(if its not properly formed).

**Comment by nkovacs** [comment]
Yes, we should document this, but we must support array messages without `msg`.

We would also put the whole array into extra, so you should see it in sentry nicely.

**Comment by albertborsos** [comment]
https://github.com/hellowearemito/yii2-sentry/commit/f95d815a73baf89ecc8454edf1aff1834f754b2e

**Comment by albertborsos** [comment]
Now with the `1.0.0` release you can log arrays with or without `msg` key.  Thanks for your feedback!

**Comment by nkovacs** [comment]
This is fixed in #13

Unfortunately, sentry doesn't display array messages nicely in the list:
![screenshot from 2017-03-27 13-06-56](https://cloud.githubusercontent.com/assets/2269242/24353547/5a9d2972-12ee-11e7-9580-e0880dc50f4d.png)

I don't know how we can improve this.
